servers/sunrpc/handshake_prog.go

```diff
@@ -6,7 +6,8 @@ import (
 	"path"
 
 	"github.com/gluster/glusterd2/utils"
-
+	"github.com/gluster/glusterd2/servers/sunrpc/program"
+	"github.com/gluster/glusterd2/servers/sunrpc/serialize"
```

Comment:
The dict serialisation is a gluster specific only. The entire dict is serialised before passing it to xdr. What xdr gets is just a opaque data blob. So technically, the dict implementation is not specific to sunrpc or xdr although that's the only place we use it today.